src/routes/auth.js

```diff
@@ -17,4 +19,11 @@ router.get(
   controllers.refreshAccessToken,
 );
 
+router.put(
+  '/:userId/change-password',
```

Comment:
I think using the id in the route is not a good idea. Is this endpoint used for one user to change his/her own password right? Then maybe instead of user id, it's better to use something like `me`, e.g. `/me/change-password`

Reply:
Actually i am not using the `id` value at all, so we can remove it. i am already extracting the `id` value from the `token`